### PR TITLE
Improved look-and-feel for mobile/tablet devices

### PR DESCRIPTION
**Pull request by donbeave**
This PR is related to that discussion: https://github.com/micronaut-projects/micronaut-core/discussions/3851.

I have improved CSS for mobile and tablet devices, here are some screenshots how it looks like on iPhone 12:

1) 

before:

CleanShot 2022-04-24 at 20 36 39@2x

after:

CleanShot 2022-04-24 at 20 46 29@2x

2)

before:

CleanShot 2022-04-24 at 20 39 59@2x

after:

CleanShot 2022-04-24 at 20 47 27@2x

CleanShot 2022-04-24 at 20 48 17@2x

3)

before:

CleanShot 2022-04-24 at 20 41 50@2x

after:

CleanShot 2022-04-24 at 20 48 59@2x

4)

before:

CleanShot 2022-04-24 at 20 44 46@2x

after:

CleanShot 2022-04-24 at 20 45 08@2x

I'm using Micronaut Security documentation as an example. You can clone this example's source code of already precompiles documentation from here: https://github.com/donbeave/micronaut-docs-improvement.

### CODE REVIEW

**Comment by sdelamo** [comment]
I will check this before 3.5.0

**Comment by donbeave** [comment]
@sdelamo sorry for pinging you, do you have time to look at this PR?

**Comment by sdelamo** [comment]
> @sdelamo sorry for pinging you, do you have time to look at this PR?

no problem for pinging. Sometimes it helps. I will review it and merge it soon. Sorry, it is taking so long.

File: src/main/docs/resources/css/custom.css [hunk 1]
```diff
@@ -427,3 +427,56 @@ html .dark-mode, body.dark-mode, .dark-mode #main, .dark-mode .docs-content {
     scrollbar-color: #555 rgba(0, 0, 0, 0.3);
     scrollbar-width: thin;
 }
+
+
+@media only screen and (max-width: 820px) {
```
Comment (timyates) [inline src/main/docs/resources/css/custom.css:432]: Elsewhere we use 768px as the threshold, ie: here

https://github.com/micronaut-projects/micronaut-docs/blob/47fb4b4e8850735315be84cbccd38fb466b62829/src/main/docs/resources/css/custom.css#L134-L141

I'm not sure which is best

**Review by timyates — APPROVED** [review]
One question about the chosen width, we seem to favor 768 as the break point, but other than consistency, I have no idea why that value was chosen originally 😀

**Review by timyates — CHANGES_REQUESTED** [review]
Actually, I built the docs for security locally using 2.0.1-SNAPSHOT of this module, and on my android phone it looks exactly the same...

@donbeave how did you test it?

File: src/main/docs/resources/css/custom.css (hunk 1 above)
Comment (donbeave) [reply src/main/docs/resources/css/custom.css:432]: 768 isn't a good choice here from my point of view, I think it is better to use viewports from the most common devices, as 768px originally was for desktop, it was not for mobile devices. I actually don't remember where I got 820px, I have changed it to this selector in the new PR:

```
@media only screen and (max-width: 428px) and (orientation: portrait), only screen and (max-width: 926px) and (orientation: landscape) {
```

This uses iPhone 13 Pro Max viewport as it is one of the biggest mobile viewports on the market: https://blisk.io/devices/details/iphone-13-pro-max

**Comment by donbeave** [comment]
@timyates you are right, I push changes to the wrong repository, please check this PR: https://github.com/micronaut-projects/micronaut-build/pull/374

I'm closing this PR as resources here is only for micronaut docs prior to 2.5.x, based on this README file: https://github.com/micronaut-projects/micronaut-docs/blob/master/src/main/docs/README.md

**Pull request closed without merging**